src/bundles/gateway.js

```diff
@@ -1,7 +1,11 @@
 import { readSetting, writeSetting } from './local-storage'
 
 export const DEFAULT_GATEWAY = 'https://ipfs.io' // TODO: switch to dweb.link when https://github.com/ipfs/go-ipfs/issues/7318
-const IMG_HASH = 'bafybeibwzifw52ttrkqlikfzext5akxu7lz4xiwjgwzmqcpdzmp3n5vnbe' // 1x1px image
+const IMG_ARRAY = [
+  { id: 'IMG_HASH_1PX', name: '1x1.png', hash: 'bafybeibwzifw52ttrkqlikfzext5akxu7lz4xiwjgwzmqcpdzmp3n5vnbe' },
+  { id: 'IMG_HASH_1PXID', name: '1x1.png', hash: 'bafkqax4jkbheodikdifaaaaabveuqrcsaaaaaaiaaaaacaidaaaaajo3k3faaaaaanieyvcfaaaabj32hxnaaaaaaf2fetstabaonwdgaaaaacsjiravicgxmnqaaaaaaiaadyrbxqzqaaaaabeuktsevzbgbaq' },
```

Comment:
> CID incompatible with DNS label length limit of 63: kdi0v5uo6z0hrybvhqyb5faspyh4pgy7cxv64dd4iwn8a1i0u0ffpn8t8960j2or6bo81cjkc2vjcrm5ebdwdl7ou91u15r1v7ywl7bv29vi2nh3t4d8o95g2sv9tzoq3evs6dk23bcfgnw5ukgfi7ngi
> 
> --- http://localhost:48084/ipfs/bafkqax4jkbheodikdifaaaaabveuqrcsaaaaaaiaaaaacaidaaaaajo3k3faaaaaanieyvcfaaaabj32hxnaaaaaaf2fetstabaonwdgaaaaacsjiravicgxmnqaaaaaaiaadyrbxqzqaaaaabeuktsevzbgbaq#x-ipfs-companion-no-redirect

I had to check that long CID. Broken on many gateways, but they all check out somewhere.

### ipfs.fleek.co gateway
* https://ipfs.fleek.co/ipfs/bafkqax4jkbheodikdifaaaaabveuqrcsaaaaaaiaaaaacaidaaaaajo3k3faaaaaanieyvcfaaaabj32hxnaaaaaaf2fetstabaonwdgaaaaacsjiravicgxmnqaaaaaaiaadyrbxqzqaaaaabeuktsevzbgbaq#x-ipfs-companion-no-redirect
* https://ipfs.fleek.co/ipfs/bafybeibwzifw52ttrkqlikfzext5akxu7lz4xiwjgwzmqcpdzmp3n5vnbe#x-ipfs-companion-no-redirect
* https://ipfs.fleek.co/ipfs/bafkreihc7efnl2prri6j6krcopelxms3xsh7undpsjqbfsasm7ikiyha4i#x-ipfs-companion-no-redirect

### ipfs.io gateway
* https://ipfs.io/ipfs/bafkqax4jkbheodikdifaaaaabveuqrcsaaaaaaiaaaaacaidaaaaajo3k3faaaaaanieyvcfaaaabj32hxnaaaaaaf2fetstabaonwdgaaaaacsjiravicgxmnqaaaaaaiaadyrbxqzqaaaaabeuktsevzbgbaq#x-ipfs-companion-no-redirect
* https://ipfs.io/ipfs/bafybeibwzifw52ttrkqlikfzext5akxu7lz4xiwjgwzmqcpdzmp3n5vnbe#x-ipfs-companion-no-redirect
* https://ipfs.io/ipfs/bafkreihc7efnl2prri6j6krcopelxms3xsh7undpsjqbfsasm7ikiyha4i#x-ipfs-companion-no-redirect